encord/objects/html_node.py

```diff
@@ -0,0 +1,69 @@
+"""
+---
+title: "Objects - HTML Node"
+slug: "sdk-ref-objects-html-node"
+hidden: false
+metadata:
+  title: "Objects - HTML Node"
+  description: "Encord SDK Objects - HTML Node."
+category: "64e481b57b6027003f20aaa0"
+---
+"""
+
+from __future__ import annotations
+
+from dataclasses import dataclass
+from typing import Collection, List, Union, cast
+
+from encord.orm.base_dto import BaseDTO
+
+
+class HtmlNode(BaseDTO):
+    """
+    A class representing a single HTML node, with the node and offset.
+
+    Attributes:
+        node (str): The xpath of the node
+        offset (int): The offset of the content from the xpath
+    """
+
+    node: str
```

Comment:
maybe we should call this `xpath`? ATM `node` is already in the class name HtmlNode, so feels like the field name can be more descriptive

Reply:
In the actual `range_html` in the `objects_index`, we call it "node" though. 

Worried it might be confusing to have it called "node" in the exports and in our DB, but then have it called "xpath" here in this one place.